planning/autoware_surround_obstacle_checker/src/node.hpp

```diff
@@ -62,27 +63,7 @@ class SurroundObstacleCheckerNode : public rclcpp::Node
 public:
   explicit SurroundObstacleCheckerNode(const rclcpp::NodeOptions & node_options);
 
-  struct NodeParam
-  {
-    bool is_surround_obstacle;
-    std::unordered_map<int, bool> enable_check_map;
-    std::unordered_map<int, double> surround_check_front_distance_map;
-    std::unordered_map<int, double> surround_check_side_distance_map;
-    std::unordered_map<int, double> surround_check_back_distance_map;
-    bool pointcloud_enable_check;
-    double pointcloud_surround_check_front_distance;
-    double pointcloud_surround_check_side_distance;
-    double pointcloud_surround_check_back_distance;
-    double surround_check_hysteresis_distance;
-    double state_clear_time;
-    bool publish_debug_footprints;
-    std::string debug_footprint_label;
-  };
-
 private:
-  rcl_interfaces::msg::SetParametersResult onParam(
```

Comment:
Where is the alternative function of `onParam`...? Or, the pickNik lib can automatically update the value?

Reply:
`ParamListener` adds a callback for each parameter internally and thus we don't have to care about updating its value, but we still need to call `get_params()` to acquire the latest values as done [here](https://github.com/mitukou1109/autoware.universe/blob/ef0b5a4835f0a49ff9cdf84971d3ad820a14dc29/planning/autoware_surround_obstacle_checker/src/node.cpp#L198).